addons/l10n_es/data/account.account.template-common.csv

```diff
@@ -340,6 +340,7 @@
 "account_common_570","Caja, euros","570","account.data_account_type_liquidity","l10n_es.account_chart_template_common","False"
 "account_common_571","Caja, moneda extranjera","571","account.data_account_type_liquidity","l10n_es.account_chart_template_common","False"
 "pgc_572_child","Bancos e instituciones de crédito c/c vista, euros","572","account.data_account_type_liquidity","l10n_es.account_chart_template_common","False"
+"account_common_572998","Cuenta transitoria","572998","account.data_account_type_current_assets","l10n_es.account_chart_template_common","False"
```

Comment:
Why adding this one manually?

Reply:
I mean, add `572999`

Reply:
When I use 572999 this account is set as Internal Transfer Account too:
![image](https://user-images.githubusercontent.com/7701001/204258565-344b9a79-a108-4897-b3f5-bc8a2028fb69.png)

If I use 572998 a new account is created from transfer_account_code_prefix and it is used as Internal Transfer Account (expected behavior):
![image](https://user-images.githubusercontent.com/7701001/204259554-62d867d3-7617-460e-a225-c61f314971ed.png)